todo_app/tests/test_trello_items.py

```diff
@@ -0,0 +1,183 @@
+from todo_app.data.trello_items import get_trello_items, get_trello_lists
+from todo_app.data.IndexViewModel import ViewModel
+from dotenv import load_dotenv, find_dotenv
+import todo_app.app as app
+
+class TestTodo:
+
+    @staticmethod
+    def test_trello_items_to_do():
```

Comment:
The intention here is for this class to be unit tests for the ViewModel class, but this relies on several Trello functions as well as Trello itself. Instead we should be creating the items dynamically, and then really tightly testing the ViewModel, for example more like:
```
        items = [
            Item('id', 'To Do Card', 'To Do', 'description', None),
            Item('id2', 'doing item', 'Doing', 'description', None),
        ]
        
        trello_lists = [
            {'name': "To Do"},
            {'name': "Doing"}
        ]
        view_model = ViewModel(items, trello_lists, True)
        ...
```

This significantly reduces the reliance of these tests on the other parts of our code, and protects them from e.g. Trello outages